test/integration/OrderBook.spec.ts

```diff
@@ -41,6 +44,16 @@ describe('OrderBook', () => {
     await orderBook.addPeerOrder(order);
   });
 
+  it('should match new ownOrder and update matches', async () => {
+    const order: orders.OwnOrder = { pairId: 'BTC/LTC', quantity: 5, price: 100 };
+    const matches = await orderBook.addOwnOrder(order);
+    expect(matches.remainingOrder).to.be.null;
+    const firstMakerLeft = await orderBookRepository.getOrderQuantity(matches.matches[0].maker);
+    expect(firstMakerLeft).to.be.equal(0);
+    const secondMakerLeft = await orderBookRepository.getOrderQuantity(matches.matches[1].maker);
+    expect(secondMakerLeft).to.be.equal(-5.5);
+  });
```

Comment:
if we want to start making this test scenario based on a sequential flow, where each step depends on the previous one's state, we should should start with an empty database (or at least with an empty `orders` table). At the moment, arbitrary dummy values from `tasks.db.init` affect this test results.

Reply:
I used the dummy values from `tasks.db.init` for this test. Should I use the values from the tests before instead?

Reply:
Yes, and please remove `tasks.db.restart` from the beginning of the tests (it's old stuff). 
Instead, you can call `db.dropTables` directly, and init some values yourself, by using the repositories objects, same as in the `init` tasks. You'll need to create some currencies and pairs, but no orders. 
You can also add `db.truncate` in the end of the tests (before `db.close`).

Reply:
Do I really need to drop all tables if I truncate them at the end of the test? And can I delete all `tasks` as they have no use anymore?
I will enable `internalmatching` for the `orderBook` in the test to use the orders created in the two tests before.

And sorry for being so sloppy and careless in this pr :disappointed: